Sources/LiveViewNative/Modifiers/Toolbars Modifiers/ToolbarModifier.swift

```diff
@@ -0,0 +1,248 @@
+//
+//  ToolbarModifier.swift
+//  LiveViewNative
+//
+//  Created by Carson Katri on 3/29/2023.
+//
+
+import SwiftUI
+import LiveViewNativeCore
+
+/// Changes the anchor point of an element in a grid.
+///
+/// Use the available toolbar elements to place items in the toolbar.
+///
+/// ```html
+/// <Text modifiers={toolbar(@native, content: "my-toolbar-content")}>
```

Comment:
After rebasing from https://github.com/liveview-native/liveview-client-swiftui/pull/734, this example should be changed to reflect namespace references changing to use atoms as of https://github.com/liveview-native/liveview-client-swiftui/pull/734/commits/2ccb51e8dde56862f6fbdee23503e2d304b682e1:

```suggestion
/// <Text modifiers={toolbar(@native, content: :my-toolbar-content)}>
```